flocker/node/agents/functional/test_cinder_behaviour.py

```diff
@@ -112,3 +127,39 @@ def test_updated_metadata_is_listed(self):
         missing_items = expected_items - actual_items
 
         self.assertEqual(set(), missing_items)
+
+
+class BlockDeviceAPIDestroyTests(SynchronousTestCase):
+    """
+    Test for ``cinder.CinderBlockDeviceAPI.destroy_volume``
+    """
```

Comment:
These tests belong in `test_cinder.py`.  Tests for behavior of our code goes there.  `test_cinder_behavior.py` is more like "hey, does Cinder actually work like we think it does?"